schlib/schlib.py

```diff
@@ -36,7 +36,7 @@ def __init__(self, filename, create = False):
 
         else:
             if not os.path.isfile(self.filename):
-                sys.stderr.write('"{filename}" is not a valid .dcm file\n'.format(filename=self.filename))
+                sys.stderr.write('"{filename}" does not exist'.format(filename=self.filename))
```

Comment:
Also: Maybe a better text:
```
  'The DCM-file for library "{filename_lib}" does not exist (expected name: "{filename}").'
```

Basically that's clear, but for newbiew its easier to talk about the .lib, as the editor mostly hides the DCM from them.